lib/lint/reporter/process-warns.js

```diff
@@ -1,5 +1,10 @@
-module.exports = function(warns, fileFilter){
+var isChildProcess = typeof process.send == 'function'; // child process has send method
+var collectFiles = require('./collectFiles');
+
+module.exports = function(flow, options){
```

Comment:
The interface was changed. 
Have you considered adding [js]docs? Like what kind of objects should be `flow` and `options` now.